tests/graphql/test-add-link-to-empty-config.graphql

```diff
@@ -0,0 +1,19 @@
+#> server-sdl
+schema @server @upstream @link(src: "tests/graphql/fixtures/link-const.graphql", type: Config) @link(src: "tests/graphql/fixtures/link-enum.graphql", type: Config) {
```

Comment:
paths should be relative to current file.

Reply:
done